adafruit_azureiot/device_registration.py

```diff
@@ -73,105 +62,102 @@ def __init__(
         self._key = key
         self._logger = logger if logger is not None else logging.getLogger("log")
 
-        requests.set_socket(socket)
+        self._mqtt_connected = False
```

Comment:
Could you instead expose MiniMQTT's `is_connected()` method?

Reply:
Done - also done in iot_mqtt.py